lib/src/lib.rs

```diff
@@ -29,4 +29,5 @@ mod streaming_body;
 mod upstream;
 mod wiggle_abi;
 
+pub use wasmtime::ProfilingStrategy;
 pub use {error::Error, execute::ExecuteCtx, service::ViceroyService, upstream::BackendConnector};
```

Comment:
style nit:

```suggestion
pub use {error::Error, execute::ExecuteCtx, service::ViceroyService, upstream::BackendConnector, wasmtime::ProfilingStrategy};
```

(_this may need to get formatted_)